python/test/Ice/custom/AllTests.py

```diff
@@ -7,12 +7,14 @@
 #
 # **********************************************************************
 
-import sys, string, re, traceback, Ice, Test
+import sys, string, re, traceback, Ice, Test, array, numpy
```

Comment:
Is numpy installed with all python installations? I don't have it with Python 3.7 installed from choco on Windows. Unless it's present with most distribution, we should test for it here. For our testing, we should install it on our VMs to make sure it's tested.